froide_evidencecollection/models.py

```diff
@@ -9,111 +9,303 @@
 from froide.publicbody.models import PublicBody
 
 
-class Source(models.Model):
-    note = models.TextField()
-    url = models.URLField(unique=True)
-    document_number = models.TextField(blank=True)
-    public_body = models.ForeignKey(PublicBody, on_delete=models.PROTECT, null=True)
-
-    def clean(self):
-        if (self.public_body and not self.document_number) or (
-            not self.public_body and self.document_number
-        ):
-            raise ValidationError(
-                "Either both or neither of document_number and public_body must be set"
-            )
-        super().clean()
+class PersonOrOrganization(models.Model):
+    external_id = models.PositiveIntegerField(
+        unique=True, verbose_name=_("external ID")
+    )
+    name = models.CharField(unique=True, max_length=255, verbose_name=_("name"))
+    affiliations = models.ManyToManyField(
+        "Institution",
+        through="Affiliation",
+        verbose_name=_("affiliations"),
+        related_name="persons",
+    )
+    regions = models.ManyToManyField(GeoRegion, verbose_name=_("regions"))
+    special_regions = ArrayField(
+        models.CharField(max_length=50),
+        default=list,
+        blank=True,
+        verbose_name=_("special regions"),
+    )
+    is_active = models.BooleanField(default=True, verbose_name=_("is active"))
+    review_comment = models.TextField(
+        default="", blank=True, verbose_name=_("review comment")
+    )
+
+    class Meta:
+        verbose_name = _("person/organization")
+        verbose_name_plural = _("persons/organizations")
 
     def __str__(self):
-        if self.document_number:
-            return f"{self.url} {self.document_number} ({self.public_body})"
-        return self.url
+        return self.name
 
-    @property
-    def domain(self) -> str:
-        return urlparse(self.url).netloc
 
+class Institution(models.Model):
+    name = models.CharField(unique=True, max_length=255, verbose_name=_("name"))
 
-class EvidenceType(models.Model):
-    name = models.TextField(unique=True)
+    class Meta:
+        verbose_name = _("institution/party level")
```

Comment:
"party level" is not applicable for a generic evidence collection app, i would suggest removing that

```suggestion
        verbose_name = _("institution")
```